src/Sarif.FunctionalTests/SarifConverterTestData/Fortify/bannedAPIs_m1.xml.sarif

```diff
@@ -1,19 +1,16 @@
 {
-  "version": "1.0.0-beta.2",
```

Comment:
i don't believe the schema version updates are buying us anything but additional work in the tree. or can you identify some value here?